Makefile

```diff
@@ -77,7 +94,7 @@ endif
 # Usage: make c package=brig test=1
 .PHONY: c
 c: cabal-fmt
-	cabal build $(WIRE_CABAL_BUILD_OPTIONS) $(package)
+	cabal build $(WIRE_CABAL_BUILD_OPTIONS) $(package) || ( make clean-hint; false )
```

Comment:
Why do you show the `clean-hint` in this specific target? :thinking:

Reply:
because i don't understand the structure of this makefile :)?  which targets would you like me to add it to?

Reply:
Ups, my mistake, sorry: `c` should be fine. (`c` is *build everything with cabal*)